common/shared.c

```diff
@@ -992,6 +1025,82 @@ ssize_t ft_inject(struct fid_ep *ep, size_t size)
 	return ret;
 }
 
+ssize_t ft_post_rma(enum ft_rma_opcodes op, struct fid_ep *ep, size_t size,
+		struct fi_rma_iov *remote, void *context)
+{
+	uint64_t cq_data = 1;
```

Comment:
Hmm, using a variable named "cq_data" could be better than just passing a number 1. :)
Anyway, I think using ft_init_cq_data seems to be a better option here.